docs/reference/commandline/build.md

```diff
@@ -300,3 +301,11 @@ like `ENV` values do.
 
 For detailed information on using `ARG` and `ENV` instructions, see the
 [Dockerfile reference](../builder.md).
+
+### Specify isolation technology for container (--isolation)
```

Comment:
@moxiegirl Table looks better, :+1: 
If I change 
```
       --isolation=                    Container isolation level
```
to
```
       --isolation=                    Container isolation technology
```

I think the flag description hard-coded in docker/docker should be updated at the same time, do you think I need to update the flag description in this commit?  Also ping @jhowardmsft